aws/table_aws_transfer_server.go

```diff
@@ -0,0 +1,260 @@
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getTransferServer,
+				Transform:   transform.FromField("PreAuthenticationLoginBanner"),
+			},
+			{
+				Name:        "post_authentication_login_banner",
+				Description: "Specifies a string to display when users connect to a server. This string is displayed after the user authenticates.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getTransferServer,
+				Transform:   transform.FromField("PostAuthenticationLoginBanner"),
+			},
+			{
+				Name:        "protocols",
+				Description: "Specifies the file transfer protocol or protocols over which your file transfer protocol client can connect to your server's endpoint.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getTransferServer,
+				Transform:   transform.FromField("Protocols"),
+			},
+			{
+				Name:        "security_policy_name",
+				Description: "Specifies the name of the security policy that is attached to the server.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getTransferServer,
+				Transform:   transform.FromField("SecurityPolicyName"),
+			},
+			{
+				Name:        "workflow_details",
+				Description: "Specifies the workflow ID for the workflow to assign and the execution role that's used for executing the workflow.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getTransferServer,
+				Transform:   transform.FromField("WorkflowDetails"),
+			},
+			{
+				Name:        "structured_log_destinations",
+				Description: "Specifies the log groups to which your server logs are sent.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getTransferServer,
+				Transform:   transform.FromField("StructuredLogDestinations"),
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Tags"),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ServerId"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Arn").Transform(arnToAkas),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listTransferServers(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// Create session
+	svc, err := TransferClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_transfer_server.listTransferServers", "client_error", err)
+		return nil, err
+	}
+	if svc == nil {
+		return nil, nil
+	}
+	// Limiting the results
+	maxLimit := int32(100)
+	if d.QueryContext.Limit != nil {
+		limit := int32(*d.QueryContext.Limit)
+		if limit < maxLimit {
+			if limit < 1 {
+				maxLimit = 1
+			} else {
+				maxLimit = limit
+			}
```

Comment:
```suggestion
			maxLimit = limit
```
If a user passes a limit value less than one, the Steampipe SDK handles it automatically, and there's no need for us to handle it explicitly.